content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,67 @@
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.  This will become the default in a future release.
+- **Added** IPv6 alpha support for Kubernetes clusters.
```

Comment:
Or even experimental.  I'm worried that this one does not have any test coverage

Reply:
doe